jt-jiffle/jt-jiffle-op/src/main/java/it/geosolutions/jaiext/jiffleop/JiffleDescriptor.java

```diff
@@ -187,8 +221,36 @@ public static RenderedOp create(RenderedImage[] sources,
         pb.setParameter(DEST_TYPE, destType);
         pb.setParameter(SRC_COORDINATE_TRANSFORMS, sourceCoordinateTransforms);
         pb.setParameter(SRC_BAND_TRANSFORMS, sourceBandTransforms);
+        pb.setParameter(DEST_BANDS, destBands);
         // JAI operation performed.
         return JAI.create("Jiffle", pb, renderingHints);
     }
 
+    /**
+     * RenderedOp creation method that takes all the parameters. Kept for backwards compatibility
+     * and deprecated, please use {@link #create(RenderedImage[], String[], String, String,
+     * Rectangle, Integer, Integer, CoordinateTransform[], BandTransform[], RenderingHints)}
```

Comment:
Does it needs an annotation?

Reply:
Let me add 2 (both javadoc and annotation)